typescript/model.ts

```diff
@@ -17,7 +17,7 @@ const metaAddressTypes = [
     "Code_aarch64",
     "Code_systemz",
 ] as const;
-type MetaAddressType = typeof metaAddressTypes[number];
+type MetaAddressType = (typeof metaAddressTypes)[number];
```

Comment:
Uh? Wrong commit?

Reply:
Must have committed it by accident, it disappeared after the rebase